src/applications/mhv-medical-records/containers/Vitals.jsx

```diff
@@ -116,7 +116,7 @@ const Vitals = () => {
         <RecordList
           records={cards}
           type={recordType.VITALS}
-          perPage={7}
+          perPage={5}
           hidePagination
         />
```

Comment:
I think this will cause Weight and Height to be pushed on to another page, but in the ticket screenshots, it looks like all the vitals remain on one page.